README.md

```diff
@@ -18,8 +18,8 @@ Version 5 is under active development. Version 4 is in support mode, and will re
 
 Version 5 upgrades the library to use ESM (ECMAScript Modules), and so your environment must fit one of the following formats to be able to use this library:
 
- * NodeJS project with `"type": "module"` in `package.json` (ESM mode)
- * Web project bundled with a tool like Webpack that can handle ESM
+-   NodeJS project with `"type": "module"` in `package.json` (ESM mode)
+-   Web project bundled with a tool like Webpack that can handle ESM
```

Comment:
Any particular reason you change these? `*` are perfectly valid markdown, as are the `|---|` (no spaces) for tables.